<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>717</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>717</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>717</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Presentation Project</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>501</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid_FileLoad" columnstretch="0,0,0,0">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="2">
      <widget class="QLineEdit" name="Line_PPTLink">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="Btn_PPTFileOpen">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>연결할 발표 자료</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>파일 저장 위치</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="Line_SaveFolder">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="Btn_SaveFolder">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>1011</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Tab_pt">
     <attribute name="title">
      <string>발표시 UI</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>421</width>
        <height>380</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <property name="horizontalSpacing">
        <number>30</number>
       </property>
       <property name="verticalSpacing">
        <number>60</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>자세에 따른 출력값 변경</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pt_Btn_OptionDialog">
         <property name="text">
          <string>
설정 변경...
</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>딥러닝 결과값</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>결과에 따른 행동</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>카메라와의 거리</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="pt_Line_DeepResult">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="pt_Line_ResultAction">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="pt_Line_CameraDistance">
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>가까움</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="pt_Bar_Distance">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="minimum">
              <number>30</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>0</number>
             </property>
             <property name="pageStep">
              <number>0</number>
             </property>
             <property name="value">
              <number>65</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>멈</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>440</y>
        <width>421</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,30,30">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="pt_Btn_Recording">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>
녹화
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pt_Btn_Start">
         <property name="text">
          <string>
시작
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pt_Btn_Stop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>
종료
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>530</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>541</height>
       </rect>
      </property>
      <property name="title">
       <string>WebCamera</string>
      </property>
      <widget class="QLabel" name="pt_WebCamera">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>530</width>
         <height>510</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Tab_ptex">
     <attribute name="title">
      <string>발표 연습시 UI</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>421</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <property name="horizontalSpacing">
        <number>30</number>
       </property>
       <property name="verticalSpacing">
        <number>40</number>
       </property>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ptex_Line_DeepResult">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ptex_Line_ResultAction">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="ptex_Line_CameraDistance">
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>가까움</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="ptex_Bar_Distance">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="minimum">
              <number>30</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>0</number>
             </property>
             <property name="pageStep">
              <number>0</number>
             </property>
             <property name="value">
              <number>65</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>멈</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>카메라와의 거리</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="ptex_Btn_OptionDialog">
         <property name="text">
          <string>
설정 변경...
</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>현재 모션</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>자세에 따른 출력값 변경</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>모션에 따른 행동</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>아이 트래킹</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="ptex_Line_ResultAction_2">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>530</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>541</height>
       </rect>
      </property>
      <property name="title">
       <string>WebCamera</string>
      </property>
      <widget class="QLabel" name="ptex_WebCamera">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>531</width>
         <height>511</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="ptex_Label_Correction">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>교정 사항</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>440</y>
        <width>421</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="10,30,30">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="ptex_Btn_Recording">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>
녹화
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ptex_Btn_Start">
         <property name="text">
          <string>
시작
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ptex_Btn_Stop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>
종료
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true">4조 MCM(미코맨)</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>aaa()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
